pom.xml

```diff
@@ -51,7 +51,7 @@
         <esh.version>0.8.0.b2</esh.version>
         <shk.version>1.2</shk.version>
         <ohc.version>2.0.0.b1</ohc.version>
-        <tp.version>2.0.0.a1</tp.version>
+        <tp.version>2.0.0.b2</tp.version>
```

Comment:
I wonder if we should completely decouple the versioning here, not to even suggest that the tp xyz belongs exactly to the openhab-distro xyz... So why not simply start with a 1.0.0?